airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/consts.py

```diff
@@ -41,6 +41,7 @@
     "airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/**",  # These files are generated and should not be formatted
     "**/airbyte-ci/connectors/metadata_service/lib/tests/fixtures/**/invalid",  # This is a test directory with invalid and sometimes unformatted code
     "airbyte-ci/connectors/pipelines/tests/test_format/non_formatted_code",  # This is a test directory with badly formatted code
+    "airbyte-integrations/connectors/destination-**/expected_spec.json",  # These files are generated by tests using Jackson's pretty printer, and should not be reformatted.
```

Comment:
kebab case instead of snake case? it's what prevails in the bulk cdk, might as well be consistent

Reply:
makes sense. I'm actually going to move this to a separate PR, pretty sure the airbyte-ci change can't be in the same PR as expected-spec.json